diff -u -r -x data HTS-demo_NIT-ATR503-M001_original/scripts/Config.pm.in HTS-demo_NIT-ATR503-M001/scripts/Config.pm.in
--- HTS-demo_NIT-ATR503-M001_original/scripts/Config.pm.in	2015-12-21 17:36:06.000000000 +0900
+++ HTS-demo_NIT-ATR503-M001/scripts/Config.pm.in	2017-04-27 14:03:24.290016931 +0900
@@ -176,7 +176,7 @@
 # modeling
 $nState      = @NSTATE@;        # number of states
 $nIte        = @NITER@;         # number of iterations for embedded training
-$beam        = '1500 100 5000'; # initial, inc, and upper limit of beam width
+$beam        = '3000 200 10000'; # initial, inc, and upper limit of beam width
 $maxdev      = @MAXDEV@;        # max standard dev coef to control HSMM maximum duration
 $mindur      = @MINDUR@;        # min state duration to be evaluated
 $wf          = @WFLOOR@;        # mixture weight flooring
